databricks_cli/sdk/service.py

```diff
@@ -52,7 +72,9 @@ def create_job(self, name=None, existing_cluster_id=None, new_cluster=None, libr
         if email_notifications is not None:
             _data['email_notifications'] = email_notifications
             if not isinstance(email_notifications, dict):
-                raise TypeError('Expected databricks.JobEmailNotifications() or dict for field email_notifications')
+                raise TypeError(
+                    'Expected databricks.JobEmailNotifications() or dict for field email_notifications'
```

Comment:
`databricks` package does not exist, neither does `JobEmailNotifications` class, so this message has to be just `Expected dict for field email_notifications`. Same for other `TypeError`s, so that it doesn't confuse developers.

Reply:
I filed https://github.com/databricks/databricks-cli/issues/508 to track.

I want to keep this PR focused on regeneration alone.